### ISSUE TITLE
Add VM WorkloadGroup to kiali Applications Dashboard

### ISSUE THREAD

**Issue by tomahkvt** [kiali/kiali#5887]
Hi Kiali Team Developers. Thank you a lot for your work.
I use
Kiali Container v1.63.2
Service Mesh Istio 1.17.0
I configurated test lab https://istio.io/latest/docs/examples/virtual-machines.
The test deployment works but I don't see anything about the VM WorkloadGroup type in Applications Dashboard.
Please add this option.
Thank you in advance.

**Comment by jshaughn** [comment]
Thanks for your feedback, @tomahkvt.  We will take a look at this request.  See also https://github.com/kiali/kiali/issues/4169, that issue is something we still need to prioritize, and I think it may include this request.

**Comment by jshaughn** [comment]
@hhovsepy , Can you review #4169 and determine whether these two issues should be consolidated?

**Comment by hhovsepy** [comment]
Yes @jshaughn those 3 issues, also [5886](https://github.com/kiali/kiali/issues/5886) should be consolidated and resolved all-together.

**Comment by hhovsepy** [comment]
Will be released in upcoming Kiali v2.5.
